.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -19,61 +19,43 @@ on:
         default: nvd-api
 
 jobs:
-  build-updater:
+  parse-versions:
     runs-on: ubuntu-latest
-    if: >
-      github.event_name == 'schedule' ||
-      (github.event_name == 'pull_request' &&
-       contains(github.event.pull_request.labels.*.name, 'pr-update-scanner-vulns'))
-    container:
-      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+    outputs:
+      versions: ${{ steps.set-versions.outputs.versions }}
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
       with:
         fetch-depth: 0
-        ref: ${{ github.event.pull_request.head.sha }}
-
-    - uses: ./.github/actions/job-preamble
-      with:
-        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
-    - uses: ./.github/actions/cache-go-dependencies
-
-    - name: Build updater
+    - name: Parse VULNERABILITY_BUNDLE_VERSION
```

Comment:
With the purpose of this job being to produce the `dev` bundle, can we remove this step entirely to skip parsing the `VULNERABILITY_BUNDLE_VERSION` file? Is there a scenario where the `dev` bundle would not be produced from master? Perhaps it would make sense to update this job directly instead of parsing the file, the meaning of `VULNERABILITY_BUNDLE_VERSION` could then be changed to represent bundles for released versions.

Reply:
Although there probably won't be too many entries in the `VULNERABILITY_BUNDLE_VERSION` (perhaps just dev, v1, v2 and v3), I just wanted to make the workflow more generic.